packages/coreutils/src/pageconfig.ts

```diff
@@ -82,6 +82,21 @@ namespace PageConfig {
     }
     return configData[name] || '';
   }
+  
+  /**
```

Comment:
Looks good, can you see about adding a test? The tests for page config are here:

https://github.com/jupyterlab/jupyterlab/blob/master/test/src/coreutils/pageconfig.spec.ts

You can build/run the tests locally with `npm run build:test` and then `npm run test`.